tests/query/insert_remove_pk.rs

```diff
@@ -35,7 +35,7 @@ fn insert_get() {
     assert_eq!(stats.primary_tables[0].n_entries, Some(1));
 
     let rw = db.rw_transaction().unwrap();
-    rw.remove(item.clone()).unwrap();
+    let _ = rw.remove(item.clone()).unwrap();
```

Comment:
This is probably a silly question, but since ```DatabaseOutputValue``` is internally a ```Vec<u8>```, how would I assert that the removed value is correct? Would I have to serialize the Item struct using bincode and compare it that way?

Reply:
My bad, I have missed this https://github.com/vincent-herlemont/native_db/pull/96#discussion_r1459419736

Reply:
I think it should be correct in the latest commit I added. I also modified the tests to assert the returned value. Let me know if there is anything else that needs to be changed.